components.yaml

```diff
@@ -51,5 +51,5 @@ components:
     url: https://github.com/k8snetworkplumbingwg/ovs-cni
     commit: 641e6348f9a6ea234e216a84784f0e1d90c80ade
     branch: release-0.36
-    update-policy: latest
+    update-policy: tagged
```

Comment:
since it is already on a branch, why do we need tagged?
unless few releases use same branch (which also doesnt make sense to do, because once both follow tagged / latest, they still have same fate)

Reply:
I want to reduce noise on stable branches, if I'm not the one maintaining them.